app_dart/lib/src/request_handlers/luci_status.dart

```diff
@@ -117,7 +116,10 @@ class LuciStatusHandler extends RequestHandler<Body> {
     if (info.expiresIn == null || info.expiresIn < 1) {
       return false;
     }
-    final ServiceAccountInfo devicelabServiceAccount = await config.deviceLabServiceAccount;
-    return info.email == devicelabServiceAccount.email;
+    final Set<String> allowedServiceAccounts = <String>{
+      'flutter-devicelab@flutter-dashboard.iam.gserviceaccount.com',
+      'flutter-dashboard@appspot.gserviceaccount.com'
```

Comment:
Can we add this in `Config`? That's where I've been adding service accounts so it's easy to reference what's being used

Reply:
This is a transitional step, I'll update the configs with the default account once we have migrated all the use cases.

Reply:
Ack